x/zoneconcierge/types/params.go

```diff
@@ -7,18 +7,23 @@ import (
 const (
 	DefaultIbcPacketTimeoutSeconds uint32 = 60 * 60 * 24       // 24 hours
 	MaxIbcPacketTimeoutSeconds     uint32 = 60 * 60 * 24 * 365 // 1 year
+	DefaultMaxHeadersPerPacket     uint32 = 100                // Max BTC headers per IBC packet
```

Comment:
One note: due to BTC retargeting interval of 2016 blocks, the base header of BTC light client contract will have to be a muliple of 2016. Then upon a handshake, Babylon might need to send over at most 2015 headers to BSN side. If we have a `MaxHeadersPerPacket` smaller than that, then in worst case the handshake might fail. But on the other hand, sending 2015 headers might be too much.

I think BTC light client contract might need some extra treatment on this, e.g., taking a vector of BTC headers upon instanitation rather than just 1 header? @KonradStaniec @liuchengxu